src/rdkafka_request.c

```diff
@@ -1878,6 +1878,49 @@ void rd_kafka_ListGroupsRequest_versioned(rd_kafka_broker_t *rkb,
         rd_kafka_broker_buf_enq_replyq(rkb, rkbuf, replyq, resp_cb, opaque);
 }
 
+/**
+ * Send DescribeGroupsRequest supporting a versioned request.
+ */
+void rd_kafka_DescribeGroupsRequest_versioned(rd_kafka_broker_t *rkb,
```

Comment:
The difference between this and the origianl function isn't that big, so I don't see a need for two different functions, just pass a max_apiversion=0 to have the old behaviour.

Reply:
done